src/main/java/Edit_Group_Profile_Use_Case/EditGroupProfileDsRequestModel.java

```diff
@@ -0,0 +1,18 @@
+package Edit_Group_Profile_Use_Case;
+
+public class EditGroupProfileDsRequestModel {
+
+    String groupName;
+    public EditGroupProfileDsRequestModel(String groupName) {
+        this.groupName = groupName;
+    }
+
+    public String getGroupName(){
+        return this.groupName;
+    }
+
+    public void setGroupName(String groupName){
+        this.groupName = groupName;
+    }
+}
```

Comment:
Since your use case is to edit the group profile, which is an attribute of the Group entity, each time you make a change we need to re-save/serialize the hashmap of groups. So instead of having the EditGroupProfileDSRequestModel, you can just use my GroupDSRequestModel since they do the same things.